tasks/README.md

```diff
@@ -67,13 +67,16 @@ Name | Summary | Category
 `task060_ropes_question_generation`	| Constructing questions regarding relations in the given paragraph.	| Question Generation
 `task061_ropes_answer_generation`	| Answering questions regarding relations in the given paragraph.	| Answer Generation
 `task062_bigbench_repeat_copy_logic`	|  Generating text that follows simple logical operations such as "repeat", "before", "after" etc.	| Logic
+`task073_CommonsenseQA_answer_generation`	|  Answering questions based on commonsense knowledge.	| Answer Generation
 `task073_CommonsenseQA_answer_generation` | Answering questions based on commonsense knowledge | Answer Generation
 `task074_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
 `task075_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation  
 `task076_splash_correcting_sql_mistakes` | Based on feedback correct the mistake in a given SQL statement. | Structured Query Generation, Text Modification
 `task077_splash_explanation_to_sql` | Generate an SQL statement based on a description of what the SQL statement does. | Structured Query Generation
 `task078_splash_sql_to_explanation` | Give a natural language description of what a given SQL statement is doing. | Structured Query Classification
 `task079_conala_concat_strings` | Given a list of strings concatenate them to form one string | Answer Generation
+`task080_piqa_answer_generation`	|  Generating solution to a goal regarding physical knowledge about the world	| Answer Generation
+`task081_piqa_wrong_answer_generation`	|  Generating incorrect solution to a goal regarding physical knowledge about the world	| Incorrect Answer Generation
```

Comment:
wrong -> incorrect

Reply:
Should I use wrong or incorrect ?  In MCTACO Tasks in the Name it is 'wrong' and in the category it is 'incorrect'.

Reply:
Alright, let's ignore my comment then.